zrml/prediction-markets/src/lib.rs

```diff
@@ -1232,6 +1299,24 @@ mod pallet {
         ValueQuery,
     >;
 
+    #[pallet::storage]
+    pub type MarketIdsPerOpenBlock<T: Config> = StorageMap<
+        _,
+        Blake2_128Concat,
+        T::BlockNumber,
+        BoundedVec<MarketIdOf<T>, ConstU32<1024>>,
+        ValueQuery,
+    >;
+
+    #[pallet::storage]
+    pub type MarketIdsPerOpenTimeFrame<T: Config> = StorageMap<
+        _,
+        Blake2_128Concat,
+        TimeFrame,
+        BoundedVec<MarketIdOf<T>, ConstU32<1024>>,
```

Comment:
```suggestion
        BoundedVec<MarketIdOf<T>, ConstU32<64>>,
```

Reply:
Mitigated in fc0555ed1d6d1c0f8c0672a47e8a96372a7db64e